.gitignore

```diff
@@ -12,8 +12,8 @@ yarn-error.log*
 storybook-static
 
 # Cypress test output videos
-apps/**/cypress/videos
-apps/**/cypress/screenshots
+sites/**/cypress/videos
+sites/**/cypress/screenshots
```

Comment:
Just change this to `**/cypress...` because ** will match 0 or more directories and I don't think we'd want to save them in one directory and not save them in others.

Reply:
Done!